### ISSUE TITLE
Tidy up codebase 🌵 

### ISSUE THREAD

**Issue by ken-futureverse** [cennznet/app-gov#26]
- [x] Stick with one way of doing conditional rendering, either use `If` component or `{if}` block
- [x] Remove `service/cennznet/src/Transaction.ts` class
- [x] Remove `node/types` and `web/types`, then migrate their contents to `node/utils` and `web/utils`
- [x] ~~Migrate all the `constants.ts` to the `apps/gov-website` as it needs to match with `env` file defined there, also remove the `.env.development`, as what requires can be inferred from the `constants.ts`~~
- [x] ~~Ensure all `library` type doesn't consume the constants directly, e.g `fetchRequiredRegistrars.ts`, `Transaction.ts`~~
- [x] Remove `node/models`
- [x] Change `"Success"` step to `"Complete"`

**Comment by aidan-starke** [comment]
If we move away from `react-extras` we can use [clsx](https://www.npmjs.com/package/clsx) for classnames

**Comment by ken-futureverse** [comment]
@aidan-starke could do that too, but I thought you are still keen to use the `Choose` component?

**Comment by aidan-starke** [comment]
yeah just throwing it out there, I think I'm getting used to ```<If>``` anyway 😅
